tests/test_bots/test_mixins/test_requests/test_chats.py

```diff
@@ -65,6 +65,11 @@ async def test_retrieving_chat_info(bot, client, message):
     assert info.group_chat_id == chat_id
 
 
+async def test_retrieving_bot_chats(bot, client, message):
+    chats = await bot.get_bot_chats(message.credentials)
+    assert len(chats) == 1
```

Comment:
chats length set not into this unit. If i change state of test-client(bot) this test fall, but feature still work.
If you testing response - get first and check type.
If you testing method - check requests to client(`client.requests[0]`)

Reply:
I really don't understand what I have to test here, because func does not expect incoming parameters like others

Reply:
Just check what the client has chat-list request.